client/src/partials/patients/documents/views/documentView.html

```diff
@@ -0,0 +1,77 @@
+<div class="row">
+  <div class="col-xs-12">
+
+    <div class="panel panel-default">
+      <!-- heading  -->
+      <div class="panel-heading clearfix">
+        <div class="btn pull-left">
```

Comment:
Do you need the "pull-left" class here?  Could you accomplish the same thing using a `<span>` without the `pull-left` class?